<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitTokenDlg</class>
 <widget class="QDialog" name="InitTokenDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Slot ID</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="mSessionText"/>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Session</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Slot list</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="mSlotIDText"/>
   </item>
   <item row="3" column="5">
    <widget class="QLineEdit" name="mLoginText"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>PIN</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="mPinText"/>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Label</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QComboBox" name="mSlotsCombo"/>
   </item>
   <item row="6" column="3" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Slot status</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Token properties</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="3">
    <widget class="QLineEdit" name="mLabelText"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InitTokenDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InitTokenDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
